### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them,  @D067796

**Review by AlexanderAlferov — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -373,7 +373,7 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"За повече информация вижте страницата „ЧЗВ“."</string>
     <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/en/faq/#encounter_but_green">"За повече информация вижте страницата „ЧЗВ“."</a></string>
+    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">"За повече информация вижте страницата „ЧЗВ“."</a></string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-bg/strings.xml:376]: This is wrong according to @janetback. All languages - but German -  should point to the English FAQ

File: Corona-Warn-App/src/main/res/values-en/strings.xml [hunk 2]
```diff
@@ -373,7 +373,7 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"For further information, please see our FAQ page."</string>
     <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/en/faq/#encounter_but_green">"For further information, please see our FAQ page."</a></string>
+    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">"For further information, please see our FAQ page."</a></string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-en/strings.xml:376]: This is wrong according to @janetback. All languages - but German - should point to the English FAQ

File: Corona-Warn-App/src/main/res/values-pl/strings.xml [hunk 3]
```diff
@@ -373,7 +373,7 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"Więcej informacji znajduje się na naszej stronie „Często zadawane pytania”."</string>
     <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/en/faq/#encounter_but_green">"Więcej informacji znajduje się na naszej stronie „Często zadawane pytania”."</a></string>
+    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">"Więcej informacji znajduje się na naszej stronie „Często zadawane pytania”."</a></string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-pl/strings.xml:376]: This is wrong according to @janetback. All languages - but German - should point to the English FAQ

File: Corona-Warn-App/src/main/res/values-ro/strings.xml [hunk 4]
```diff
@@ -373,7 +373,7 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"Pentru mai multe informații, consultați pagina noastră de întrebări frecvente."</string>
     <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/en/faq/#encounter_but_green">"Pentru mai multe informații, consultați pagina noastră de întrebări frecvente."</a></string>
+    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">"Pentru mai multe informații, consultați pagina noastră de întrebări frecvente."</a></string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-ro/strings.xml:376]: This is wrong according to @janetback. All languages - but German - should point to the English FAQ

File: Corona-Warn-App/src/main/res/values-tr/strings.xml [hunk 5]
```diff
@@ -373,7 +373,7 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"Daha fazla bilgi için lütfen SSS sayfamıza bakın."</string>
     <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/en/faq/#encounter_but_green">"Daha fazla bilgi için lütfen SSS sayfamıza bakın."</a></string>
+    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">"Daha fazla bilgi için lütfen SSS sayfamıza bakın."</a></string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-tr/strings.xml:376]: This is wrong according to @janetback. All languages - but German - should point to the English FAQ

**Review by ralfgehrer — APPROVED** [review]

**Pull request merged**